.github/workflows/e2e-nvidia-t4-x1.yml

```diff
@@ -107,11 +107,6 @@ jobs:
           python3.11 -m pip install packaging wheel setuptools-scm
           python3.11 -m pip install .[cuda]
 
-      - name: ilab system info
-        run: |
-          . venv/bin/activate
-          ilab system info
```

Comment:
Perhaps, instead of removing this helpful-when-debugging-a-failure command, it can be moved into the next step before running the tests.

Reply:
https://github.com/instructlab/instructlab/blob/main/scripts/basic-workflow-tests.sh#L410